tox.ini

```diff
@@ -1,16 +1,33 @@
 [tox]
 envlist = py27,py36,py37,black
 
-[testenv]
+[testenv:py27]
 passenv = CIRCLECI CIRCLE_* CI_PULL_REQUEST
 extras =
   tests
   docs
 commands =
   pytest --cov-report=xml --cov=mirdata {toxinidir}/tests/
 
-[black]
-basepython = py37
+[testenv:py36]
+passenv = CIRCLECI CIRCLE_* CI_PULL_REQUEST
+extras =
+  tests
+  docs
+commands =
+  pytest --cov-report=xml --cov=mirdata {toxinidir}/tests/
+
+[testenv:py37]
+passenv = CIRCLECI CIRCLE_* CI_PULL_REQUEST
+deps = black
```

Comment:
we probably dont need this if [testenv:black] is doing it instead.

Reply:
💯